type-definitions/index.d.ts

```diff
@@ -19,6 +19,8 @@ declare interface DisposeFn {
   (): void | Promise<any>;
 }
 
-export default function create<A>(f: (add: (a: A) => void,
+declare function create<A>(f: (add: (a: A) => void,
                                       end: (x?: A) => void,
                                       error: (e: Error) => void) => void | DisposeFn ): Stream<A>;
+
+export = create;
```

Comment:
My understanding is, the issue comes from the fact that this dist of this module uses a CommonJS-oriented default export which is not semantically supported in ES6.  Typescript gets around this via the `export = blah ` + `import blah = require(...)` syntax.  I may be wrong and there may be a more elegant way.  I needed this to compile for a PR I was making elsewhere so when this change fixed the compile error, I did not research further.